core/meta/src/TClass.cxx

```diff
@@ -664,7 +664,7 @@ void TDumpMembers::Inspect(TClass *cl, const char *pname, const char *mname, con
          snprintf(&line[kvalue],kline-kvalue,"->0");
       else if (!isbasic) {
          if (!fNoAddr) {
-            snprintf(&line[kvalue],kline-kvalue,"->%lx ", (Long_t)p3pointer);
+            snprintf(&line[kvalue],kline-kvalue,"->%lx ", (Longptr_t)p3pointer);
```

Comment:
```suggestion
            snprintf(&line[kvalue],kline-kvalue,"->%llx ", (Long64_t)p3pointer);
```
since `Longptr` is either a long or an int.

Reply:
Oh yes, right, thanks!

Reply:
I will have plenty of the same kind of change coming soon (I first fix the compilation errors on Win64, then the warnings, that are real errors actually)